### PR TITLE
feat: Add dashboard state to error telemetry to better debug errors

### PR DESCRIPTION
**Pull request by AdityaHegde**
closes https://github.com/rilldata/rill-private-issues/issues/511

### CODE REVIEW

File: web-common/src/metrics/ErrorEventHandler.ts [hunk 1]
```diff
@@ -133,6 +142,12 @@ export class ErrorEventHandler {
       screenName,
       stack,
       message,
+      dashboardState,
     ]);
   }
+
+  private getDashboardState() {
+    // TODO: we need to also save other params that get added after
```
Comment (ericpgreen2) [inline web-common/src/metrics/ErrorEventHandler.ts:150]: What about just storing the whole URL? The main goal is to make it easier to reproduce bugs, so it'd be great if we can just copy/paste the URL into our browser.

Comment (AdityaHegde) [reply web-common/src/metrics/ErrorEventHandler.ts:150]: Ya i thought we save org & project. I guess not, so it makes sense to do this

**Review by ericpgreen2 — APPROVED** [review]
Good from the frontend POV. @esevastyanov, can you please augment the data pipeline with this new `pageURL` field?

**Review by nishantmonu51 — APPROVED** [review]

**Comment by nishantmonu51** [comment]
merging it, Lets create a separate ticket for the pipeline work.

**Comment by esevastyanov** [comment]
Created a [ticket](https://github.com/rilldata/rill-private-issues/issues/525)

**Pull request merged**